src/cmd.rs

```diff
@@ -10,19 +10,15 @@ use termcolor::Color::Green;
 ///
 pub fn call(command: &[&str], path: &str, shell: &mut Shell, dry_run: bool) -> Result<bool, Error> {
     if dry_run {
-        try!(
-            shell
-                .status_with_color("", format!("cd {}", path), Green)
-                .map_err(SyncFailure::new)
-        );
-        try!(
-            shell
-                .status_with_color("", format!("{}", command.join(" ")), Green)
-                .map_err(SyncFailure::new)
-        );
-        try!(shell.status_with_color("", "cd -", Green).map_err(
+        shell
+            .status_with_color("", format!("cd {}", path), Green)
+            .map_err(SyncFailure::new)?;
```

Comment:
This is pretty hard to spot. Is `try!` one of those things that is going to hang around or is it dead? Because I can see that in some places `?` is a clear winner and some other times we might perfer to be louder and use `try`.

Reply:
Do you think we can codify this into a rule we can generally apply? For instance, I don't think this particular one's potential failure matters a great deal because it's just about printing to the console.

The actual win from using `?` is that it is polymorphic over some trait (which I believe is internal at the moment), that is implemented by both `Option` and `Result` and makes it easier to swap out the implementation without having to change the consumer side.

Reply:
Was making a general observation with this comment, not a specific action item for this line. The `?` sounds like a good improvement. Just wish it was easier to see. It's already approved so feel free to merge this.

Reply:
Cool. We're definitely on the same page here.